modules/nodes-nodes-rebooting-gracefully.adoc

```diff
@@ -47,13 +69,43 @@ $ systemctl reboot
 ----
 +
 In a moment, the node enters the `NotReady` state.
++
+[NOTE]
+====
+With some SNO clusters, the `oc` commands might not be available after you cordon and drain the node because the `openshift-oauth-apiserver` pod is not running. You can use SSH to connect to the node and perform the reboot.
+
+[source,terminal]
+----
+$ ssh core@<master-node>.<cluster_name>.<base_domain>
+----
+
+[source,terminal]
+----
+$ sudo systemctl reboot
+----
+====
 
-. Mark the node as schedulable after the reboot is complete:
+. After the reboot is complete, mark the node as schedulable:
```

Comment:
Worth updating this procedure with the "by running the following command" intro for commands? Maybe not since it's not going into `main`. Totally fine if that's out of scope :)